linux-tools/patchutils/patchutils.sh

```diff
@@ -56,7 +63,12 @@ function tc_local_cleanup()
 function run_test()
 {
 pushd $PATCHUTILS_TESTS_DIR >$stdout 2>$stderr
-TST_TOTAL=`ls tests -I common.sh -I soak-test | wc -l`
+grep -i "ubuntu" /etc/*-release >/dev/null 2>&1
```

Comment:
"grep -i "ubuntu" /etc/*-release" is being called multiple times in multiple scripts
declare DISTRO in a shared function and use it all over here